### PR TITLE
Add the 'confetti prank box' project.

### PR DESCRIPTION
**Pull request by ganicke**
There also is a video demonstrating the prank. I can add that when someone pops it onto the youtube channel.

The pics are somewhat overexposed. I hate to make them do retakes. Need to keep the flash off.

### CODE REVIEW

File: docs/projects/confetti-box.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Materials 
+
+### Electronics
+* Adafruit Circuit Playground Express
+* Battery holder and 3 AAA batteries
+* 180° rotation servo
+* 3 crocodile clips to male connector wires (alternatively simply connect crocodile clips to wires)
+* USB cable
+
+### New stuff
```
Comment (pelikhan) [inline docs/projects/confetti-box.md]: What is "new stuff"?

**Comment by pelikhan** [comment]
We have a lightbox at the office to take the pictures. Can we retake them? The quality is quite poor on those.

**Comment by riknoll** [comment]
If we do retake them, let's use the servo from the educators pack

**Comment by pelikhan** [comment]
We should retake the pictures as part of testing the activity using our light box.

**Comment by Jaqster** [comment]
Yeah, these photos are not good. Also, I think we should try building it ourselves to verify the steps, and that it actually works.

File: docs/projects/confetti-box.md (hunk 1 above)
Comment (ganicke) [inline docs/projects/confetti-box.md]: Stuff that's not recycled, new supplies, regular supplies. 😵

I'll put it back under Materials.

**Comment by ganicke** [comment]
Changed out old washed out images with better ones.

**Comment by audreyctseng** [comment]
changed some sentences grammatically // fixed spelling errors - I'll start building this project today and see how far I can get - I'll retake pictures along the way as well @ganicke @pelikhan

**Comment by pelikhan** [comment]
Awesome!

**Pull request merged**